plugin-gradle/README.md

```diff
@@ -659,6 +659,16 @@ Note that `enforceCheck` is a global property which affects all formats (outside
 
 <a name="examples"></a>
 
+## Can I apply Spotless to specific files?
+
+You can target specific files by setting the `files` project property to a comma-separated list of file patterns:
+
+```
+cmd> gradlew spotlessApply -Pfiles=my/file/pattern.java,more/generic/.*-pattern.java
```

Comment:
Suggestion, can we namespace this argument?
```
gradlew spotlessApply -PspotlessFiles=my/file/pattern.java,more/generic/.*-pattern.java
```
That way this significantly reduces the chances of a collision with another plugin or a user's own build logic.

Reply:
good idea. updated